app/services/ExactTargetService.scala

```diff
@@ -226,8 +226,10 @@ class ExactTargetService(
 }
 
 object SqsClient extends LazyLogging {
-  private val sqsClient = new AmazonSQSClient(CredentialsProvider)
-  sqsClient.setRegion(Region.getRegion(Regions.EU_WEST_1))
+  private val sqsClient = AmazonSQSClient.builder
```

Comment:
yeah I always struggle with formatting for chained stuff - usually for me the answer is "don't" but I can see why in this case it's not meaningful to split things in any other way.